dali/daliadmin/daliadmin.cpp

```diff
@@ -1889,16 +1889,32 @@ static void listmatches(const char *path, const char *match, const char *pval)
     {
         output.append(", match=").append(match);
         if (pval)
-            output.append(", property value = ").append(pval);
+            output.append(", property value(s) = ").append(pval);
     }
+    outln(output);
```

Comment:
Is this an extra outln() ?

Reply:
Yes, it was unintentionally missing before, meaning the lines above it were building up a string ready for output, that never was.